snapcraft/storeapi/__init__.py

```diff
@@ -91,6 +92,10 @@ def __init__(self, conf, root_url):
         self.conf = conf
         self.root_url = root_url
         self.session = requests.Session()
+        # Setup max retries for all store URLs and the CDN
+        self.session.mount('http://', HTTPAdapter(max_retries=5))
+        self.session.mount('https://', HTTPAdapter(max_retries=5))
```

Comment:
from the error in https://travis-ci.org/snapcore/snapcraft/jobs/209838628 and given --debug I know it works somehow.... we can make it wait a few seconds, that would be a tad more complex, want me to do that?